dashboard/app/models/user.rb

```diff
@@ -210,7 +211,9 @@ class User < ApplicationRecord
   has_many :pd_workshops_organized, class_name: 'Pd::Workshop', foreign_key: :organizer_id
 
   has_many :authentication_options, dependent: :destroy
+  accepts_nested_attributes_for :authentication_options
   belongs_to :primary_contact_info, class_name: 'AuthenticationOption', optional: true
+  accepts_nested_attributes_for :primary_contact_info
```

Comment:
I'm actually not sure we need this. We should test it, but I would assume that because primary_contact_info is just an authentication_option, line 214 takes care of this for us. But maybe that's just me hoping for too much out of Rails/Active Record

Reply:
We can definitely remove this. We ended up not storing the `:primary_contact_info` in the session cache, so this is not needed.